.babelrc

```diff
@@ -1,4 +1,12 @@
 {
-  "presets": [ "@babel/env", "@babel/preset-react"],
-  "plugins": [ "react-hot-loader/babel"]
+  "presets": [
+    "@babel/env",
+    "@babel/preset-react"
+  ],
+  "plugins": [
+    "react-hot-loader/babel"
+  ],
+  "scripts": {
```

Comment:
I dont think we need this in the web pack config, this should be in the package Json.  if you would like to load the mocha module for web pack,  take a peak [here](https://webpack.js.org/loaders/mocha-loader/). for now lets remove this.

Reply:
Just took it out. It worked fine without it.